test_project/app/api/endpoints/shop_api.py

```
@@ -90,6 +93,65 @@ async def get_categories(
+        )
+
+        employee = await Employee.async_queryset.with_transaction(transaction).create(
+            department_id=department.id,
+            name=payload.get('employee_name', 'Committed Employee'),
+            email=payload.get('employee_email', 'committed.employee@example.com'),
+            salary=payload.get('employee_salary', 75000),
+            hire_date=date.today(),
+        )
+    except Exception as exc:
+        await transaction.rollback()
+        raise exc
```

Ruff (0.14.7)

Use `raise` without specifying exception name

Remove exception name

(TRY201)